ci/steps/docker-buildx-push.sh

```diff
@@ -24,7 +24,7 @@ main() {
   fi
 
   # Download the release-packages artifact
-  download_artifact release-packages ./release-packages
+  download_artifact release-packages ./release-packages "production" "v$VERSION"
```

Comment:
🧹 chore: download_artifact now takes 4 args. Although we set this as default fallback values when calling download_artifact I prefer to explicitly set them here so there's no guessing.